modulefiles/ufs_noaacloud.intel_debug.lua

```diff
@@ -0,0 +1,20 @@
+help([[
+loads UFS Model prerequisites for cloud/intel
+]])
+
+prepend_path("MODULE_PATH", "/contrib/spack-stack/envs/ufs-wm/install/modulefiles/Core")
+
+hpc_intel_ver=os.getenv("hpc_intel_ver") or "2021.3.0"
+load(pathJoin("stack-intel", hpc_intel_ver))
+
+hpc_impi_ver=os.getenv("hpc_impi_ver") or "2021.3.0"
+load(pathJoin("stack-intel-oneapi-mpi", hpc_impi_ver))
+
+load("ufs_common_spack")
```

Comment:
shouldn't this to point to a debug version of the common modules?

Reply:
@ulmononian I believe it should. The plan is to use Mark's spack installation in these lua files so I assume there are two installations, with one for debug?